src/main/java/uk/ac/ebi/ena/webin/cli/service/SampleService.java

```diff
@@ -20,18 +21,30 @@
 import org.springframework.web.client.ResourceAccessException;
 import org.springframework.web.client.RestTemplate;
 
+import uk.ac.ebi.biosamples.client.model.auth.AuthRealm;
+import uk.ac.ebi.biosamples.client.service.WebinAuthClientService;
 import uk.ac.ebi.ena.webin.cli.service.exception.ServiceException;
 import uk.ac.ebi.ena.webin.cli.service.exception.ServiceMessage;
 import uk.ac.ebi.ena.webin.cli.utils.RetryUtils;
 import uk.ac.ebi.ena.webin.cli.validator.reference.Sample;
 
+import java.net.URI;
+import java.util.Arrays;
+
 public class 
 SampleService extends WebinService

```

Comment:
Unrelated comment regarding existing URL variable naming in WebinService. Rename these:
```
    private final static String TEST_URL = "https://wwwdev.ebi.ac.uk/ena/submit/drop-box/";
    private final static String PRODUCTION_URL = "https://www.ebi.ac.uk/ena/submit/drop-box/";
    private final static String TEST_SUBMISSION_URL = "https://wwwdev.ebi.ac.uk/ena/submit/webin-v2/";
    private final static String PRODUCTION_SUBMISSION_URL = "https://www.ebi.ac.uk/ena/submit/webin-v2/";
```
to
```
```
    private final static String WEBIN_REST_V1_TEST_URL = "https://wwwdev.ebi.ac.uk/ena/submit/drop-box/";
    private final static String WEBIN_REST_V1_PRO_URL = "https://www.ebi.ac.uk/ena/submit/drop-box/";
    private final static String WEBIN_REST_V2_TEST_URL = "https://wwwdev.ebi.ac.uk/ena/submit/webin-v2/";
    private final static String  WEBIN_REST_V2_PROD_URL = "https://www.ebi.ac.uk/ena/submit/webin-v2/";
```